### PR TITLE
don't show PG stuff while editing course_info

### PR DESCRIPTION
**Pull request by Alex-Jordan**
When editing `course_info`, this suppress the PG help stuff at the top, as well as the selects for problem seed and display mode.

### CODE REVIEW

File: lib/WeBWorK/ContentGenerator/Instructor/PGProblemEditor.pm [hunk 1]
```diff
@@ -233,6 +233,9 @@ sub initialize ($c) {
 	$c->stash->{formsToShow}      = ACTION_FORMS();
 	$c->stash->{actionFormTitles} = ACTION_FORM_TITLES();
 
+	# Tell the templates if we are working on a PG file
+	$c->{is_pg} = ($c->{file_type} eq 'course_info') ? 0 : 1;
```
Comment (drgrice1) [inline lib/WeBWorK/ContentGenerator/Instructor/PGProblemEditor.pm:237]: Why do this?  There is nothing wrong with checking the value of `$c->{file_type}` in the templates.  This is unnecessary.

Comment (Alex-Jordan) [reply lib/WeBWorK/ContentGenerator/Instructor/PGProblemEditor.pm:237]: In the hardcopy-theme branch, there is a new file type `hardcopy_theme`. All of the places that check for `'course_info'` will now be checking two things. Making this change now will just make the diff for that branch cleaner. But if you prefer I can leave it all in that branch.

Comment (drgrice1) [reply lib/WeBWorK/ContentGenerator/Instructor/PGProblemEditor.pm:237]: I see.  I suppose it will be okay here then.

**Comment by Alex-Jordan** [comment]
I fixed my typos in the commit message.

**Review by drgrice1 — APPROVED** [review]
This is good.  Those things are certainly not needed for a course info file.

**Review by pstaabp — APPROVED** [review]

**Pull request merged**